spec/features/staff/users_can_choose_recipient_country_spec.rb

```diff
@@ -55,28 +55,11 @@
         expect(page).to have_selector "li.autocomplete__option", text: "Afghanistan", visible: true
       end
 
-      scenario "typing a complete country name, clicking it in the list and clicking continue saves the country" do
-        fill_in I18n.t("form.label.activity.recipient_country"), with: "Saint Lucia"
-        find("li.autocomplete__option", text: "Saint Lucia").click
-        click_button I18n.t("form.button.activity.submit")
-        click_on I18n.t("default.link.back")
-        click_on I18n.t("tabs.activity.details")
-
-        within(".recipient_country") do
-          expect(page).to have_content "Saint Lucia"
-        end
-      end
-
       scenario "typing a partial match, clicking on the complete match and clicking continue saves the country " do
         fill_in I18n.t("form.label.activity.recipient_country"), with: "saint"
         find("li.autocomplete__option", text: "Saint Lucia").click
         click_button I18n.t("form.button.activity.submit")
-        click_on I18n.t("default.link.back")
-        click_on I18n.t("tabs.activity.details")
-
-        within(".recipient_country") do
-          expect(page).to have_content "Saint Lucia"
-        end
+        expect(activity.reload.recipient_country).to eq("LC") # Saint Lucia
```

Comment:
Although we could make a back-end assertion like this I think we could get even more value by checking that the full string "Saint Lucia" appears on the page. It'd remove the need for the comment too. Related to the previous comment about how we could do this now the back buttons are gone.